### ISSUE TITLE
Add support for likes

### ISSUE THREAD

**Issue by jamesvandyne** [jamesvandyne/tanzawa#110]
I didn't think I'd want "like" posts, but I'm finding I want to interact with sites without bookmarking or replying. Not sure if these should appear in the main feed?

**Comment by rmdes** [comment]
on the Known/Idno indieweb implementation, each instance can be configured to decide what types of posts appear in the "front page" - I would usually hide the like/replies from the front for example to make it cleaner, less noisy. 

Also what about Like syndication ? let's say I "like" a tweet from tanzawa, does it syndicate to twitter and gets the tweet liked by the account associated with the blog ?

**Comment by jamesvandyne** [comment]
Yes - right now you can achieve "hiding from the front page" for replies with setting the post permission to "People who know the url". I imagine likes would be similar.

Once Tanzawa gets native twitter syndication support (via a plugin), I think it would be natural to have it also handle syndicating the like as well.
